templates/taxi/driver_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1>Driver List <a href="{% url "taxi:driver-create" %}">+</a>
```

Comment:
The URL tag uses double quotes, which can cause issues in some cases. Consider using single quotes for consistency: `{% url 'taxi:driver-create' %}`.